cli/tools/test/reporters/pretty.rs

```diff
@@ -331,7 +368,10 @@ impl TestReporter for PrettyTestReporter {
     _tests: &IndexMap<usize, TestDescription>,
     _test_steps: &IndexMap<usize, TestStepDescription>,
   ) {
-    common::report_summary(&self.cwd, &self.summary, elapsed);
+    if self.repl && self.summary.total == 0 {
+      return;
+    }
```

Comment:
Are we gonna try to run tests on every expression evaluation in the REPL?

Reply:
> Are we gonna try to run tests on every expression evaluation in the REPL?

It doesn't do that much work but I guess it adds two extra `.await`ed calls on every eval. I'll explicitly check for tests in the repl instead.